CakePHP/tests/ruleset.xml

```diff
@@ -1,52 +0,0 @@
-<?xml version="1.0"?>
```

Comment:
@ADmad I might be wrong but this was only the ruleset for the old testsuite,  the main ruler is still here. I didn't try to customize it but I think it is still possible